src/Navigation/WPF/Navigation.WPF/Service/NavigationService.cs

```diff
@@ -80,8 +83,15 @@ protected NavigationStackEntry Current
 
 
         public IViewModel RootViewModel => Root?.ViewModel;
+
         public IViewModel CurrentViewModel => Current?.ViewModel;
 
+
+        public bool CanGoBack => BackStack?.Any() == true;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unnecessary Boolean literal(s).](https://app.codacy.com/gh/UltimateCodeMonkeys/CodeMonkeysMVVM/pullRequest?prid=5674091)